tests/test_compression.py

```diff
@@ -0,0 +1,19 @@
+from scitbx.array_family import flex
+
+from dxtbx.ext import compress, uncompress
+
+
+def test_compress_decompress():
+    x, y = 10, 10
+
+    original = [1 for n in range(x * y)]
+
+    original[10] = 44369
+    original[11] = 214
+
+    data = flex.int(original)
```

Comment:
```suggestion
    data = flex.int(x * y, 1)
    data[10] = 44369
    data[11] = 214
```
You can construct a `flex.int` full of `1`s directly.